tests/unit_tests/test_wave_06.py

```diff
@@ -260,8 +273,9 @@ def test_swap_best_by_category_no_other_match_is_false():
         my_priority="Electronics",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    #Assert
+    assert result == False
```

Comment:
Writing some additional assertions here to check that Tai and Jesse's inventory has not changed would make this test more robust.